src/test/unit/DelegationUnit.t.sol

```diff
@@ -2689,40 +2685,41 @@ contract DelegationManagerUnitTests_Undelegate is DelegationManagerUnitTests {
 
         // try to call the `undelegate` function and check for reversion
         cheats.prank(caller);
-        cheats.expectRevert("DelegationManager.undelegate: operators cannot be undelegated");
+        cheats.expectRevert(IDelegationManager.OperatorsCannotUndelegate.selector);
         delegationManager.undelegate(defaultOperator);
     }
 
     //TODO: verify that this check is even needed
-    function test_undelegate_revert_zeroAddress() public {
-        _registerOperatorWithBaseDetails(defaultOperator);
-        _delegateToOperatorWhoAcceptsAllStakers(address(0), defaultOperator);
-
-        cheats.prank(address(0));
-        cheats.expectRevert("DelegationManager.undelegate: cannot undelegate zero address");
-        delegationManager.undelegate(address(0));
-    }
-
-    /**
-     * @notice Verifies that the `undelegate` function has proper access controls (can only be called by the operator who the `staker` has delegated
-     * to or the operator's `delegationApprover`), or the staker themselves
-     */
-    function testFuzz_undelegate_revert_invalidCaller(
-        address invalidCaller
-    ) public filterFuzzedAddressInputs(invalidCaller) {
-        address staker = address(0x123);
-        // filter out addresses that are actually allowed to call the function
-        cheats.assume(invalidCaller != staker);
-        cheats.assume(invalidCaller != defaultOperator);
-        cheats.assume(invalidCaller != defaultApprover);
+    // function test_undelegate_revert_zeroAddress() public {
+    //     _registerOperatorWithBaseDetails(defaultOperator);
+    //     _delegateToOperatorWhoAcceptsAllStakers(address(0), defaultOperator);
```

Comment:
Why is this one commented out?

Reply:
It looks like the cancelSalt() no longer reverts if the provided salt has already been cancelled.